### ISSUE TITLE
Update OSM wiki and possibly Wikipedia with relevant info

### ISSUE THREAD

**Issue by nyurik** [maplibre/maplibre-gl-js#3934]
We need to go through the public information available on OSM wiki to make sure it is all relevant and up to date. We may also need to search Wikipedia to double check the information there. This is more of a tracking ticket to remind us to do it, and to coordinate these efforts.

**Comment by HarelM** [comment]
Can you clarify what wikipedia etc related to maplibre you want to check?
Did you mean to open this issue for maputnik?

**Comment by nyurik** [comment]
No, i did mean to open it here (although I did consider `maplibre/maplibre` too). There are several aspects of this ticket:
* Make sure OSM wiki has the right references to MapLibre, together with some basic documentation/examples if needed
* Make sure OSM wiki lists MapLibre together with all other similar libraries and projects
* EN Wikipedia currently mentions MapLibre in a few places, but it might be good idea to submit MapLibre as a draft article, and work with the community to make it into a proper article

**Comment by nyurik** [comment]
P.S. This ticket could be extended to collaborate with all external user-contributed sites, like responding to [MapLibre-GL](https://stackoverflow.com/questions/tagged/maplibre-gl) stackoverflow questions

**Comment by HarelM** [comment]
Should I assign this issue to you then?
